src/main/java/seedu/address/model/medicalhistory/Timestamp.java

```diff
@@ -2,17 +2,27 @@
 
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
 import java.time.format.FormatStyle;
+import java.util.Locale;
 
 /**
  * Timestamp class used in the {@code Diagnosis} class.
  * Provides an immutable timestamp of when a diagnosis was created.
  */
 public class Timestamp {
+
     public static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter
-            .ofLocalizedDateTime(FormatStyle.MEDIUM, FormatStyle.SHORT);
+            .ofLocalizedDateTime(FormatStyle.MEDIUM, FormatStyle.SHORT)
+            .withLocale(new Locale("en", "SG"));
+    /**
+     * Format pattern of {@code DATE_TIME_FORMATTER}, follows pattern specified by java.time.format.DateTimeFormatter.
+     */
+    public static final String DATE_TIME_FORMATTER_PATTERN = "d mmm yyyy, h:mm a";
```

Comment:
Consider standardising date-time format. Refer to Appointment.java

Reply:
The date-time used here provides better readability for doctors viewing the medical history of patients. Whereas for the Appointment class, the ease of input by using a numerical format of "dd-MM-yyyy HH:mm" may be preferred.